pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -921,6 +923,24 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		}
 	}
 
+	// Append HostDevices to DomXML if Nvidia GPU requested
+	if util.IsGpuVmi(vmi) {
+		vgpuPciAddresses := append([]string{}, c.VgpuDevices...)
```

Comment:
Please rename this variable to reflect that this is a list of mdev uuids

Reply:
Updated